R/benchTradePerf.R

```diff
@@ -0,0 +1,200 @@
+#' Trade Execution Performance Benchmarks
+#' 
+#' This function gathers different benchmarking methods used to evaluate the average execution price \eqn{\bar{P}} of a given trading strategy.
+#' The \eqn{\bar{P}} is compared against a number of benchmark metrics, in order to assess its performance in terms of profit or loss.
+#' These methods are not strictly alternative, indeed each of them provides different insights and suffers limitations.
+#' 
+#' The performance is quantified by means of a \emph{Profit and Loss (PnL) metric}. 
+#' A positive PnL metric indicates that the trading strategy outperformed a chosen benchmark on average,
+#' viceversa negative values register an underperformance.
+#' 
+#' By and large, PnL metrics are computed as:
+#' 
+#' 
+#' where \eqn{P_{avg}} is the average execution price and \eqn{P_{B}} is a given benchmark price that can be a single next/present open/close price.
+#' It is worth stressing that they are expressed in basis points (bps) units.
+#' 
+#' A widely used one is the \emph{Volume Weighted Average Price (VWAP) benchamark}... 
+#' 
+#' A variation of the VWAP benchmark is given by the \emph{Participation Weighted Price (PWP) benchmark}.
+#' 
+#' 
+#' Lastly the \emph{Relative Performance Measure}, which differs from the PnL metrics above. It is expressed as:
+#' 
+#' 
```

Comment:
Yes. In reality, as I written in the PR opening comment all the documentation other than the code related one is a TODO at the moment. Although it will be documented, there is much book text in the issue and a vignette will be written as well.